test/tests/api-cit/v2_0/sel_alert_poller_tests.py

```diff
@@ -268,9 +268,9 @@ def test_full_sel(self):
 
         sel_file = self.__create_selEntries_file(available_sel_entries)
         fit_common.remote_shell('ls')
-        fit_common.scp_file_to_host(sel_file)
+        sel_file_path = fit_common.scp_file_to_host(sel_file)
 
-        self.__run_ipmitool_command(bmc_ip, "sel add ~/selError.txt")
+        self.__run_ipmitool_command(bmc_ip, "sel add {0}".format(sel_file_path))
```

Comment:
sel_file_path is parsed by os.path.basename, you may want "~/{}".format(sel_file_path)